src/pages/Search/DesktopResults.tsx

```diff
@@ -29,6 +29,13 @@ function DesktopResults({ results }: { results: CareProviderSearchResult[] }) {
     rerenderMap();
   }, [results]);
 
+  useEffect(() => {
```

Comment:
what does this do? can we add a comment to the code about the what/why here?

Reply:
done 👍